### DISCUSSION TITLE
Application crahses on libcef.dll

### DISCUSSION THREAD

**Discussion by GitNewbieHub** [cefsharp/CefSharp#4595] [General]
I'm upgrading CefSharp from 49.0.1.0 to 109.1.11 for one of my Winform project(dll). Everything works fine locally. I copied my updated dll file and all the CefSharp related files to a test server(Windows Server 2012R2. I'm getting application has stopped error

Faulting application name: MyApp.Client.exe, version: 22.4.0.0, time stamp: 0x6510f3ee
Faulting module name: libcef.dll, version: 109.1.11.0, time stamp: 0x63b8965c
Exception code: 0x4000001f
Fault offset: 0x01587b28
Faulting process id: 0x254
Faulting application start time: 0x01d9f09d35ab820c
Faulting application path: C:\Program Files (x86)\MyApp\MyApp.Client.exe
Faulting module path: C:\Program Files (x86)\MyApp\\libcef.dll
Report Id: 9b061e52-5c90-11ee-8198-000d3a4632ed
Faulting package full name: 
Faulting package-relative application ID: 

Error occurred on following line code 

Cef.Initialize(settings);

Here is the log from debug.log

[0926/195836.784:ERROR:dxva_video_decode_accelerator_win.cc(1460)] DXVAVDA fatal error: could not LoadLibrary: msmpeg2vdec.dll: The specified module could not be found. (0x7E)
[0926/195836.784:ERROR:dxva_video_decode_accelerator_win.cc(1468)] DXVAVDA fatal error: could not LoadLibrary: msvproc.dll: The specified module could not be found. (0x7E)

I copied the two files from my Windows 10 machine to the server's system32 folder but it didn't resolve the error. Also tried different version(109.0.9.0), same issue. I couldn't find much info on google also.

**Comment by amaitland** [comment]
Duplicate of #4593

Please add detail to your existing discussion rather than creating multiple.

**Reply by GitNewbieHub** [reply]
I downgraded the version from 116.0.230.0 due to the Windows Server 2012 to 109.1.11 and ran into different error which is this thread about. 109.1.11 should works on Windows Server 2012.

**Reply by GitNewbieHub** [reply]
I will add the details to other thread. Thank!!
